action.yml

```diff
@@ -0,0 +1,82 @@
+name: 'build-singularity-cvmfs-action'
+description: 'GitHub Action Package for Requesting Singularity Builds on CVMFS'
+branding:
+  icon: 'upload-cloud'
+  color: 'blue'
+
+inputs:
+  github_token:
+    description: 'a PAT so the bot can git push to WIPACrepo/cvmfs-actions'
+    required: true
+    default: ''
+  remove:
+    description: 'whether to request to remove the images(s); use `${{ github.event_name == 'delete' }}`'
+    required: true
+    type: boolean  # the incoming type is boolean, but to access it's a string; e.i. 'true', 'false'
+    default: 'false'
+  docker_tags:
+    description: 'the list of Docker image tags requested to be built on CVMFS; use `${{ needs.docker.outputs.tags }}`'
+    required: true
+    type: string
+    default: ''
+
+
+# outputs:
+#   random-number:
+#     description: "Random number"
+#     value: ${{ steps.random-number-generator.outputs.random-id }}
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'  # NOTE: update, someday
+
+    - name: Checkout WIPACrepo/cvmfs-actions
+        uses: actions/checkout@v3
+        with:
+          repository: WIPACrepo/cvmfs-actions
+          token: ${{ inputs.github_token }}  # so job can git push
+
+    - name: Git config
+      run: |
+        git config user.name github-actions
+        git config user.email github-actions@github.com
+      shell: bash
+
+    - name: Request to Update Image(s) on CVMFS
+      run: |
+        if ${{ inputs.remove == 'false' }}; then
+          ACTION=add
+        elif ${{ inputs.remove == 'true' }}; then
+          ACTION=remove
+        else
+          echo "ERROR: Unsupported action: ${{ inputs.remove }}"
+        fi
+
+        for TAG in ${{ fromJSON(inputs.docker_tags) }}; do
+          echo $TAG
+          python3 ${{ github.action_path }}/update_docker_images_txt.py \
+              --action $ACTION \
+              --docker-tag $TAG \
+              --dest-dir realtime \
```

Comment:
`dest-dir` should not be hard-coded, and should instead be part of the inputs.  A default is probably `None` or the empty string, so cvmfs is a mirror of the docker tag.  The realtime case is somewhat special, I think.

Reply:
Good point